packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1295,39 +1302,6 @@ describe("Runtime", () => {
 				);
 			});
 
```

Comment:
Can you add some tests that validate entry point works as expected?

Reply:
There are unit tests that validate what gets passed into `loadRuntime(...)` is what's returned by `getEntryPoint()`. Not sure what else we would test beyond that.

Reply:
Okay, that's basically what I was looking for - getEntryPoint() returns as per what's in initializeEntryPoint().